src/pods/test-sentences/test-sentences.container.tsx

```diff
@@ -50,14 +47,41 @@ export const TestSentencesContainer: React.FC = () => {
     );
   }, []);
 
+  const pickRandomVerb = (selectedVerbs: string[]): VerbEntityApi => {
+    if (Array.isArray(selectedVerbs)) {
+      const selectedVerbsLength = selectedVerbs.length;
+      const index = Math.floor(Math.random() * selectedVerbsLength);
+      const [selectedVerbWithInfo] = verbCollection.filter(
+        verb => verb.infinitive === selectedVerbs[index]
+      );
+      return selectedVerbWithInfo;
+    }
+    return {
+      infinitive: '',
+      past: '',
+      participle: '',
+      translation: '',
+    };
+  };
+
+  const randomVerb = pickRandomVerb(selectedVerbs);
```

Comment:
¿Is this executed on every redraw? Is a little bit strange, once you refactor the other two methods, the UI component will be smaller and it will be easier to check how does this work

Reply:
done.